### PR TITLE
feat(//tests): Update rtol and atol based tolerance for test cases

### PR DESCRIPTION
**Pull request by andi4191**
Signed-off-by: Anurag Dixit <a.dixit91@gmail.com>

# Description
Updating the tolerance metric using absolute and relative tolerance threshold.

This PR directly impacts the CI, DLFW workflow.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
#1054 

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by andi4191** [comment]
Current threshold in test cases will act as absolute tolerance limit as intended.

File: tests/util/util.cpp [hunk 1]
```diff
@@ -5,13 +5,14 @@ namespace torch_tensorrt {
...
 
-bool almostEqual(const at::Tensor& a, const at::Tensor& b, float threshold, float atol = 1e-8, float rtol = 1e-5) {
+bool almostEqual(const at::Tensor& a, const at::Tensor& b, float atol = 1e-8, float rtol = 1e-5) {
   LOG_GRAPH(a << std::endl << b << std::endl);
   auto a_float = a.toType(at::kFloat);
   auto b_float = b.toType(at::kFloat);
 
   auto diff = a_float - b_float;
-  auto result = diff.abs().max().item<float>() - (atol + rtol * b.abs().max().item<float>());
+  auto result = diff.abs().max().item<float>();
```
Comment (narendasan) [inline tests/util/util.cpp:14]: This equation is still confusing (see #1052)

**Review by peri044 — APPROVED** [review]
LGTM according to the equation that torch.allclose is using

**Pull request merged**